crates/turbopack/src/module_options/mod.rs

```diff
@@ -404,40 +430,7 @@ impl ModuleOptions {
                             ReferenceType::Css(CssReferenceSubType::AtImport),
                         )),
                     ]),
-                    [
-                        if let Some(options) = enable_postcss_transform {
-                            let execution_context = execution_context.context(
-                                "execution_context is required for the postcss_transform",
-                            )?;
-
-                            let import_map = if let Some(postcss_package) = options.postcss_package
-                            {
-                                package_import_map_from_import_mapping(
-                                    "postcss".to_string(),
-                                    postcss_package,
-                                )
-                            } else {
-                                package_import_map_from_context("postcss".to_string(), path)
-                            };
-                            Some(ModuleRuleEffect::SourceTransforms(Vc::cell(vec![
-                                Vc::upcast(PostCssTransform::new(
-                                    node_evaluate_asset_context(
-                                        execution_context,
-                                        Some(import_map),
-                                        None,
-                                        "postcss".to_string(),
-                                    ),
-                                    execution_context,
-                                )),
-                            ])))
-                        } else {
-                            None
-                        },
-                        Some(ModuleRuleEffect::ModuleType(ModuleType::CssGlobal)),
-                    ]
-                    .into_iter()
-                    .flatten()
-                    .collect(),
+                    [ModuleRuleEffect::ModuleType(ModuleType::CssGlobal)],
```

Comment:
```suggestion
                    vec![ModuleRuleEffect::ModuleType(ModuleType::CssGlobal)],
```

Reply:
removed the `.collect()` after testing 🙈